webapp/pom.xml

```diff
@@ -407,5 +400,40 @@
         </plugins>
       </build>
     </profile>
+
+    <profile>
+      <id>jms</id>
+      <!-- Adds JMS-based EventService implementation-->
+      <dependencies>
+        <dependency>
+          <groupId>org.trellisldp</groupId>
+          <artifactId>trellis-jms</artifactId>
+          <version>${trellis.version}</version>
+        </dependency>
+        <dependency>
```

Comment:
Does `trellis-jms` not have a dependency on `trellis-event-serialization`? @acoburn should it have?

Reply:
No, there's no direct dependency. The service loader just tries to load some implementation of `ActivityStreamService` (`trellis-event-serialization` is one such implementation). This makes it possible to swap that out if you want some different type of serialization.